Makefile

```diff
@@ -2,7 +2,7 @@
 LOCALE_VERSION=0.2
 CPU_MITIGATIONS_VERSION=0.3
 PROMETHEUS_VERSION=0.3.1
-PROMETHEUS_EXPORTERS_VERSION=0.9.0
+PROMETHEUS_EXPORTERS_VERSION=1.0.0
```

Comment:
I am just curious, why `1.0.0` and not `0.10.0`? 

Not that I am complaining, just trying to understand the logic for the versioning :-)

Reply:
I've introduced the new schema for the formula data json object and this change is not backwards compatible.

Reply:
Ok, does this affect the SUSE Manager customers/Uyuni users in any way, or is this transparent for them?

Should we maybe mention this at the changelog for those using this standalone?

Reply:
The change is transparent. The script migrates the existing formula data in the background.

I've added information about lacking backwards compatibility to the changelog.